src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -292,9 +362,22 @@ limitations under the License.
         The Current Weekly Schedule Command is sent from the server in response to the Get Weekly Schedule Command.
       </description>
       <arg name="NumberOfTransitionsForSequence" type="int8u"/>
-      <arg name="DayOfWeekForSequence" type="DayOfWeek"/>
-      <arg name="ModeForSequence" type="ModeForSequence"/>
-      <arg name="Transitions" type="ThermostatScheduleTransition" array="true"/>
+      <arg name="DayOfWeekforSequence" type="ScheduleDayOfWeekBitmap" min="0x0" max="0xFF"/>
+      <arg name="ModeForSequence" type="ScheduleModeBitmap" min="0x0" max="0x3"/>
+      <arg name="Transitions" type="WeeklyScheduleTransitionStruct" array="true" length="10"/>
+    </command>
+    <command source="server" code="0x01" name="GetRelayStatusLogResponse" optional="true">
```

Comment:
This command is not part of Matter and should not be here.  See "9.2.5. Element Requirements" in the device library spec.

Reply:
hmmm...not sure how that got in there, that's from the updated incoming spec I believe.

Reply:
@hasty They should not have done that.  I agree that this "oh, let's override in the device library" thing is pretty broken.  :(